test/general/key.js

```diff
@@ -3934,4 +4028,48 @@ XvmoLueOOShu01X/kaylMqaT8w==
     expect(signingKeySignature instanceof openpgp.SignaturePacket).to.be.true;
     expect(signingKeySignature.keyFlags[0] & openpgp.enums.keyFlags.signData).to.be.equals(openpgp.enums.keyFlags.signData);
   });
+
+  it("Sign and verify key - key with user attribute", async function () {
+    let publicKey = await openpgp.readKey({ armoredKey: keyWithImageData });
+    const privateKey = await openpgp.decryptKey({
+      privateKey: await openpgp.readKey({ armoredKey: priv_key_rsa }),
+      passphrase: "hello world",
+    });
+
+    const { minRSABits } = openpgp.config;
+    openpgp.config.minRSABits = 1024;
+    try {
+      publicKey = await publicKey.signAllUsers([privateKey]);
+      const signatures = await publicKey.verifyAllUsers([privateKey]);
+      const publicSigningKey = await publicKey.getSigningKey();
+      const privateSigningKey = await privateKey.getSigningKey();
+      expect(signatures.length).to.equal(4);
+      expect(signatures[0].userID).to.equal(publicKey.users[0].userID.userID);
+      expect(signatures[0].userAttribute).to.be.null;
+      expect(signatures[0].keyID.toHex()).to.equal(
+        publicSigningKey.getKeyID().toHex()
+      );
```

Comment:
This is very nitpicky but just for consistency with those ones, could you format these as one line?
(This formatting arguably does look nicer but it visually separates the `.valid` checks from the other ones, which is slightly confusing.)

Reply:
Indeed, it's way better in a single line. thank you 😄